app/assets/javascripts/admin/addon/services/color-palette-change-tracker.js

```diff
@@ -0,0 +1,33 @@
+import { tracked } from "@glimmer/tracking";
+import Service from "@ember/service";
+import { TrackedSet } from "@ember-compat/tracked-built-ins";
+
+export default class ColorPaletteChangeTracker extends Service {
```

Comment:
I'm not sure if this qualifies as an Ember service which is an Ember object that lives for the duration of the application. In the case of this usage here, I think defining the tracked properties on the controller is a better fit?

Reply:
Ok moved the logic to the controller (it didn't occur to me that the controller instance is available to the anonymous component in my route template as `@controller`/`this.args.controller` so I thought using the controller wasn't possible).